incubator/tensorflow-inception/values.yaml

```diff
@@ -0,0 +1,35 @@
+## TensorFlow Serving server image version
+## ref: https://hub.docker.com/r/bitnami/tensorflow-serving/tags/
+##
+server:
+  image: tompizmor/tensorflow-serving:latest
+  port: 9000
+
+## TensorFlow Serving Client image version
+## ref: https://hub.docker.com/r/bitnami/tensorflow-serving-client/tags/
+##
+client:
+  image: tompizmor/tensorflow-serving-client:latest
```

Comment:
I think you meant to use `bitnami/tensorflow-serving-client`, but I don't see it exist https://hub.docker.com/r/bitnami/tensorflow-serving-client